BuildTools/Nuspecs.UWP/Package.UniversalWindowsPlatform.nuspec

```diff
@@ -1,24 +1,26 @@
 ﻿<?xml version="1.0"?>
 <package >
-    <metadata>
-        <id>$Id$</id>
-        <version>$Version$</version>
-        <title>$Title$</title>
-        <authors>Progress</authors>
-        <owners>Progress</owners>
-        <projectUrl>http://www.telerik.com/universal-windows-platform-ui</projectUrl>
-        <licenseUrl>https://github.com/telerik/UI-For-UWP/blob/master/LICENSE.md</licenseUrl>
-        <iconUrl>http://www.telerik.com/docs/default-source/nuget-feed-product-files/nugeticon_uwp.png?sfvrsn=2</iconUrl>
-        <requireLicenseAcceptance>true</requireLicenseAcceptance>
-        <description>UI for Universal Windows Platform is a toolset for building Universal Windows Platform apps for the Windows Store and the enterprise. The library is designed to offer the same user experience, functionality and behavior on Windows devices of all form factors.</description>
-		<releaseNotes>For full release notes see https://github.com/telerik/UI-For-UWP/releases/tag/1.0.2.7</releaseNotes>
-		<tags>UWP Windows Telerik Controls XAML C#</tags>
-        <language>en-US</language>
-    </metadata>
-    <files>
-        <file src="$DeployDirectory$\$BinariesSubDir$\**" target="lib\uap10.0"/>
-        <file src="$NuspecsDir$\$Id$.targets" target="build\uap10.0"/>
-		<file src="$NuspecsDir$\VisualStudioToolsManifest.xml" target="tools"/>
-        <file src="$DeployDirectory$\License.md" target="LicenseAgreements" />
-    </files>
+  <metadata>
+    <id>$Id$</id>
+    <version>$Version$</version>
+    <title>$Title$</title>
+    <authors>Progress</authors>
+    <owners>Progress</owners>
+    <projectUrl>http://www.telerik.com/universal-windows-platform-ui</projectUrl>
+    <license type="file">LicenseAgreements\LICENSE.md</license>
+    <iconUrl>http://www.telerik.com/docs/default-source/nuget-feed-product-files/nugeticon_uwp.png?sfvrsn=2</iconUrl>
+    <icon>images\uwp.png</icon>
```

Comment:
The path is images\uwp.png, but the file is added into BuildTools\Nuspec.UWP\uwp.png. Can you double-check this, please?

Reply:
1. BuildTools/BuildNuget.UWP.proj has been formatted this is the only change.
2. In the nuspec file under section  <files> is indicated from where the file to be taken and further saved under "images" folder in the nuget package.
<file src="$NuspecsDir$\uwp.png" target="images\" />

$NuspecsDir$ = BuildTools\Nuspec.UWP\uwp.png

Metadata element  <icon>images\uwp.png</icon> is used by nuget package itself to know from where to get the icon in order to show it.